ui/src/components/UserProfile/UserProfile.jsx

```diff
@@ -4,21 +4,40 @@ import axios from "axios"
 import * as config from '../../config'
 import * as Bootstrap from "react-bootstrap"
 import 'bootstrap/dist/css/bootstrap.min.css';
+import { useState, useEffect } from "react"
 
 
 export default function UserProfile({ user }) {
+    const [cities, setCities] = useState([])
+
     // Contains the users information in a Bootstrap Card component
+    React.useEffect(() => {
+        let setup = async()=>{
+            const response = await axios.get(`http://localhost:3001/cities/`).catch((err)=>{
```

Comment:
In a followup diff, you can take the error message and show a warning message on the screen if there's an issue loading the cities.

Reply:
Adding to next PR